src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -307,9 +307,12 @@ def helm_install_release(chart_path, subscription_id, kubernetes_distro, kuberne
                         "--set", "systemDefaultValues.clusterconnect-agent.enabled=true",
                         "--output", "json"]
     # Add custom-locations related params
-    if enable_custom_locations:
+    if enable_custom_locations and (enable_private_link is None or enable_private_link == "false"):
         cmd_helm_install.extend(["--set", "systemDefaultValues.customLocations.enabled=true"])
         cmd_helm_install.extend(["--set", "systemDefaultValues.customLocations.oid={}".format(custom_locations_oid)])
+    # Disable cluster connect if private link is enabled
+    if enable_private_link == "true":
+        cmd_helm_install.extend(["--set", "systemDefaultValues.clusterconnect-agent.enabled=false"])
```

Comment:
With cluster connect disabled, custom location params are moot. Should we add warning if cx has explicitly passed cl params. Also we can disable CL feature.

Reply:
Found that you are already showing that warning if prompt is not disabled. However we dont call out that oid will be ignored.

Reply:
Ack. Will add that part in prompt message.

Reply:
added a warning message for this